include/node.hpp

```diff
@@ -478,14 +478,11 @@ class merged_node : public node<merged_node<Left, Right, OutId, InId>, meta::con
     }
 
     template<typename... Ts>
-    // In order to have nicer error messages, this is checked in the function body
-    // requires input_port_types::template are_equal<std::remove_cvref_t<Ts>...>
+    // Nicer error messages for the following would be good, but not at the expense of breaking
+    // can_process_simd.
+        requires(input_port_types::template are_equal<std::remove_cvref_t<Ts>...>)
```

Comment:
Does this imply that all input port types have to be equal? 

Haven't checked this before but this would be a stronger constraint since there are cases where int-float, float/double, or float/std::complex input port types are mixed.

Reply:
No, it tests that each input port type is equal to the corresponding type in the pack `Ts`.

Reply:
Thanks!